rdkcentral/Dobby
# Add breakpad support

### Description
Adds breakpad support to Dobby using `breakpad_wrapper` (https://wiki.rdkcentral.com/display/RDK/Integrating+Breakpad+to+RDK-V+Component)

### Test Procedure
During build, ensure that CMake find breakpad. E.G
```
-- Found BREAKPAD: /home/developer/rdk/rdk-workspace/build-arrisxi6/tmp/work/cortexa15t2hf-neon-rdk-linux-gnueabi/dobby/3.0-r0/recipe-sysroot/usr/lib/libbreakpadwrapper.so  
```

If breakpad is found, then breakpad support is automatically enabled.

When DobbyDaemon starts, breakpad will print the minidump location:
```
2021 Jun 10 13:26:26.605716 DobbyDaemon[2470]: Breakpad: minidump location: [/opt/secure/minidumps/2fa2e5f2-0c03-412e-72c134a5-1e706fd8.dmp]
```

On the STB, force a crash of DobbyDaemon (`kill -11 {pid of DobbyDaemon}`) and ensure that a minidump is created

### Type of Change

### Requires Bitbake Recipe changes?

To support breakpad, add 
```
DEPENDS += "breakpad breakpad-wrapper"
```
to the Dobby recipe.
